### PR TITLE
completed javascript-if exercise

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
GxC0Csq

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  },
+
+  biden: {
+    name: 'Joe Biden',
+    age: 98472347892374
+  }
+};
+
+function isOldEnoughToDrink(person) {
+  return people[person].age >= 21;
```
Comment (codingcodymiller) [inline javascript-if/main.js:35]: You shouldn't be using `people` here, just `person`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 1 above)
Comment (joseph-w-valdez) [reply javascript-if/main.js:35]: i'll fix it

**Comment by joseph-w-valdez** [comment]
Rd0iA3g

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+function isUnderFive(number) {
+  return number < 5;
+}
+
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+function startsWithJ(string) {
+  return string.split('')[0] === 'J';
```
Comment (codingcodymiller) [inline javascript-if/main.js:19]: `split` is not necessary here

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by joseph-w-valdez** [comment]
xpttjGV

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**